src/patternfly/components/DataList/docs/data-list-simple-h2.md

```diff
@@ -0,0 +1,4 @@
+### Usage
+| Class | Applied to | Outcome |
+| ------ | -- | -- |
+| `.pf-c-data-list__cell` > *    | `h2`  | Provides better separation of list items by defining the primary content section as a heading. Easy way for screen reader users to navigate to specific sections on the page. Heading level should be based on the context in which the DataList component is being used. E.g., if the heading for the section that contains the DataList is a level 3, then `h4` elements should be used in the DataList items. **Required** |
```

Comment:
These tables are primarily used to describing what classes or attributes are can or should be used in the component. But this example is just illustrating how the heading element can be incorporated into the datalist html structure.

I would remove the table and just have a paragraph that explains why headings are beneficial. And then the end of the explanation can include a statement like "Refer to the [example label] for an example of how a heading can be used in this component.

Also remove the "**Required**". This is used to indicate classes or attributes that result in the component not being usable or rendered as expected.

Reply:
I take back my suggestion to include this statement, "Refer to the [example label] for an example of how a heading can be used in this component."

Sometimes our component documentation is included at the end of the page, but in this case it's not and is adjacent to the example. So we can leave that statement out.